README.md

```diff
@@ -1 +1,99 @@
-# book-dirt-bike-back-end
\ No newline at end of file
+# Dirt Bikes API
+
+> Dirt Bikes is an application to book dirt bikes for rent. The user will be able to choose a bike according to his preferences and book a period during which he will use the desired bike.
+
+<h2 align="center">
+  Demo 📝
+</h2>
+
+![image](/app/assets/images/WhatsApp%20Image%202022-09-29%20at%2014.17.39.jpeg)
+
+
+## Built With
+
+- Ruby
+- Rails
+- PostgreSQL
+- Rspec
+- JwT
+- Device
+
+
+**Live**
+
+- Live app: [Dirt-Bike](https://incredible-unicorn-5f0c7d.netlify.app/)
+
+## Front-end repo
+[Frontend repo](https://github.com/Tufahel/book-dirt-bike-front-end)
+
+
+## API Documentation
+[API DOC](https://calm-brushlands-14039.herokuapp.com/api-docs/index.html)
+
+
+## Getting Started
+
+To get a local copy up and running follow these setup steps.
+
+**Setup**
+
+- Open terminal and clone the project: `git clone git@github.com:Tufahel/book-dirt-bike-back-end.git`
+- cd into the directory `book-dirt-bike-back-end`
+
+**Prerequisites**
+
+- You need to have `node.js` installed in your system
+
+**Install**
+
+- Clone the repository to your local machine
+- Run `bundle install` to install the necessary gems
+- Run `rails db:create` to create the database
+- Run `rails db:migrate` to migrate the database
+- Run `rails db:seed` to seed the database
+- Run `rails s` to start the server
+- Open `http://localhost:3000/` in your browser
+- In case you run into any errors please raise an [issue](https://github.com/Tufahel/book-dirt-bike-back-end/issues).
+
+### Run tests
+
+- Install Rspec with `rails generate rspec:install`
+- Run `rspec spec` to run the tests
+
+## Authors
+
+👤 **Toyosi Taiwo**
+
+- GitHub: [@Yorsyboy](https://github.com/Yorsyboy)
+- Twitter: [@tboytaiwo](https://twitter.com/Tboytaiwo)
+- LinkedIn: [Toyosi Taiwo](https://linkedin.com/in/taiwo-toyosi)
+
+👤 **Tufahel Ahmed**
+
+- GitHub: [@Tufahel](https://github.com/Tufahel)
+- Twitter: [@TufahelAhmed](https://twitter.com/TufahelAhmed)
+- LinkedIn: [@Tufahel-Ahmed](https://www.linkedin.com/in/tufahel-ahmed/)
+
+👤 **Nii Amanor Djoleto**
+
+- Github: [@Ramrod433](https://github.com/ramrod433)
+- Twitter: [@_Djoleto](https://twitter.com/_djoleto_)
+- LinkedIn: [Nii Djoleto](https://www.linkedin.com/in/nii-amanor-djoleto)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Credits
+
+Design inspiration: [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Your project lacks a license file and a license section on your readme.  Please, kindly add the license file and corresponding readme section with a link to it. 

You can go [here]https://github.com/microverseinc/readme-template and grab the MIT.md file, put it at the root of your project, and link to it on your readme.

Reply:
This is a very important thing for a professional project. Thank you very much @Meltrust for suggesting this change.